Sources/ApodiniDatabase/Handlers/Delete.swift

```diff
@@ -0,0 +1,20 @@
+import Fluent
+import Apodini
+@_implementationOnly import Vapor
+
+/// A `Handler` that deletes the object with the given `IDValue` in the database, if it exists. If not an error is thrown.
+/// It uses the database that has been specified in the `DatabaseConfiguration`.
+public struct Delete<Model: DatabaseModel>: Handler {
+    @Apodini.Environment(\.database)
+    private var database: Fluent.Database
+
+    @Parameter(.http(.path))
+    var id: Model.IDValue
+    
+    public func handle() -> EventLoopFuture<UInt> {
+        Model.find(id, on: database)
+            .unwrap(orError: Abort(.notFound) )
+            .flatMap { $0.delete(on: database ) }
+            .transform(to: HTTPStatus.ok.code )
```

Comment:
Not sure if I'm missing something, but it's currently not possible to return `EventloopFuture<Void>` as a `Response`, because Void can't conform to `ResponseTransformable`

Reply:
Ah, I see. We should probably add an implementation to transform `Void` to a response `.final(Void())`. I would suggest to create an issue about this to discuss if the decisions to return a `.final(Void())` would be correct. An alternative would be to return `.nothing` but this would not create any response to the client which would be needed here, therefore I would not suggest to use `.nothing` here.

Reply:
Ah, I see. We should probably add an implementation to transform `Void` to a response `.final(Void())`. I would suggest to create an issue about this to discuss if the decisions to return a `.final(Void())` would be correct. An alternative would be to return `.nothing` but this would not create any response to the client which would be needed here, therefore I would not suggest to use `.nothing` here.